### DISCUSSION TITLE
How to expand to two depNames on match?

### DISCUSSION THREAD

**Discussion by akselleirv** [renovatebot/renovate#17750] [Request Help] [answered]
Hello everyone!

I'm using CUE to define my k8s definisions, and I'm using a variable to create a string based on the configured environment.

Let's say I have this string:
```
image: "myregistry.com/my-service/\(cfg.cluster.env):1.0.0"
``` 
The `cfg.cluster.env` can expand to either `production` or `staging`.

I'm using the regexManagers I have this matchString `image: \"(?<depName>myregistry\.com.*):(?<currentValue>.*)\"`
Which matches the value, but it will return HTTP 404 as the container does not exist.

Is it possible to tell Renovate that if it matches `\(cfg.cluster.env)`, it should expand it to:
- `myregistry.com/my-service/production:1.0.0`
- `myregistry.com/my-service/staging:1.0.0`

**Comment by rarkins** [comment, marked as answer]
Tried just having two regex manager definitions? One expands to staging and one to production?

**Reply by akselleirv** [reply]
Can you point me in a direction of how to expand?

**Reply by rarkins** [reply]
Use depNameTemplate

**Reply by akselleirv** [reply]
Great, thank you!
